Libraries/NativeAnimation/RCTNativeAnimatedModule.m

```diff
@@ -144,7 +154,7 @@ - (dispatch_queue_t)methodQueue
   [_activeAnimations addObject:animationDriver];
   _animationDrivers[animationId] = animationDriver;
   [animationDriver startAnimation];
-  [self startAnimation];
+  [self startAnimationLoopIfNeeded];
```

Comment:
@janicduplessis - as discussed on Saturday, this should be `[self updateAnimation]` :) for others reading this, this is important because 1) it updates the value right away, rather than potentially waiting a frame 2) we don't repeatedly startup and teardown loop which, combined with 1) sometimes led to a race condition where the value that started the loop would never get applied because another would arrive before it had a chance to.